Included Kite as a supported connectivity platform (#408)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/connectivity.md

The Connectivity agent, which works from within the {{< product-c8y-iot >}} Device Management application, provides basic information on mobile devices and additional connectivity details.

> The SIM connectivity platforms supported by {{< product-c8y-iot >}} are Comarch, Ericsson, Jasper and Kite.

Reviewer: I would actually have "Jasper" in front. But we do not know if Kite is an integration or experimental.

> The SIM connectivity platforms supported by {{< product-c8y-iot >}} are Comarch, Ericsson, Jasper and Kite.

Reviewer: ```suggestion
The {{< product-c8y-iot >}} platform integrates with the SIM connectivity platforms Comarch, Ericsson, Jasper and Kite.
```

> {{< product-c8y-iot >}} integrates with the SIM connectivity management platform Jasper. For the SIM connectivity management platforms Comarch and Ericsson, {{< product-c8y-iot >}} provides an experimental implementation. For more details, please contact [product support](/welcome/contacting-support/).

Reviewer: @l2p- Where should we put Kite in this introductory text?

Reviewer: > @l2p- Where should we put Kite in this introductory text? Is it an integration like Jasper or experimental like Comarch and Ericsson.

Author: @l2p- Can you kindly respond? Thanks a lot.

> {{< product-c8y-iot >}} integrates with the SIM connectivity management platform Jasper. For the SIM connectivity management platforms Comarch and Ericsson, {{< product-c8y-iot >}} provides an experimental implementation. For more details, please contact [product support](/welcome/contacting-support/).

Reviewer: ```suggestion
```

* Enter the ICCID in the **Info** tab, then click **Save**.
* Click **Reload** in the top menu bar to make the **Connectivity** tab appear.

> >**Info:** Note that it may take a few seconds until the tab appears for the first time on a device, as {{< product-c8y-iot >}} checks if the particular SIM card is managed by the SIM provider.

Reviewer: This should use admonitions here and below.

Author: I included the admonitions in the PR 438. This is so that I can cherry-pick this PR to the other releases.

Reviewer: Good point. But this will definitely end up in a conflict later which needs to be carefully resolved taking into consideration both changes.
